troncos/traces/__init__.py

```diff
@@ -100,6 +132,9 @@ def init_tracing_provider(
     if global_provider:
         opentelemetry.trace.set_tracer_provider(provider)
 
+    if os.environ.get("TRACING_DEBUGGING_FILE"):
+        init_tracing_debug(provider)
```

Comment:
```suggestion
```

It's strange to add debugging here only if you specify `TRACING_DEBUGGING_FILE` but if you don't, you have to call `init_tracing_debug` yourself. I vote removing this.